src/types/ui.ts

```diff
@@ -60,22 +60,23 @@ export type OracleQueryUI = {
   project: Project;
   title: ReactNode;
   description: ReactNode;
-  identifier: string;
   expiryType: ExpiryType | null;
   oracleAddress: Address;
+  // not always available with assertions
+  identifier?: string;
```

Comment:
@md0x I think the subgraph schema for v3 is wrong here. The schema says the `identifier` is non-nullable, but for v3 it can be empty

Reply:
@ryanwolhuter its possible this will always be available from the graph, but when generating state from an event, its not gauranteed to exist. we may only have a settle or dispute event, which will not include much of the info. the only thing gauranteed from events is the `assertionId`.

Reply:
Yeah I say this because I can see assertions with null for the identifier coming from the graph, on goerli at least.